web_api/core/test_stripe_views.py

```diff
@@ -30,22 +105,162 @@ def test_stripe_self_serve_redirect_view(mocker: MockFixture) -> None:
         return_value=fake_billing_portal_session,
     )
 
-    user = User.objects.create(
-        github_id=10137,
-        github_login="ghost",
-        github_access_token="33149942-C986-42F8-9A45-AD83D5077776",
-    )
-    account = Account.objects.create(
-        github_installation_id=377930,
-        github_account_id=900966,
-        github_account_login=user.github_login,
-        github_account_type="User",
-    )
-    AccountMembership.objects.create(account=account, user=user, role="member")
+    user, account = create_account()
 
     client = Client()
     client.login(user)
     res = client.get(f"/v1/t/{account.id}/stripe_self_serve_redirect")
 
     assert res.status_code == 302
     assert res["Location"] == fake_billing_portal_session.url
+
+
+@pytest.mark.django_db
+def test_get_subscription_info_view_valid_account_personal_user(
+    mocker: MockFixture,
+) -> None:
+    """
+    all personal accounts have valid subscriptions
+    """
+    user, account = create_account()
```

Comment:
Shouldn't this test have active users? Anyone with zero active users is going to have "VALID_SUBSCRIPTION".